File: tasks/textOverlayOnHeroImage.cpp
std::vector<int> textOverlayOnHeroImage(std::vector<std::vector<int> > image, int height, int width) {
  int sum;
  int maxx=0;
  std::vector<int> r(2,0);
  for(int i=0; i+height<=image.size(); i++) {
    for(int j=0; j+width<=image[0].size(); j++) {
      sum=0;
      for(int k=0; k<height; k++) {
        for(int m=0; m<width; m++) {
          sum+=image[i+k][j+m];
        }
      }
      if(sum>maxx) {
        r[0]=i;
        r[1]=j;
        maxx=sum;
      }
    }
  }
  return r;
}

File: tasks/textOverlayOnHeroImage.js
function textOverlayOnHeroImage(image, height, width) {

  var max = 0;
  var coords = [];
  for(var i = 0; i <= image.length - height; i++) {
    for(var j = 0; j <= image[0].length - width; j++) {
      var brightness = 0;
      for(var y = i; y < i + height; y++) {
        for(var x = j; x < j + width; x++) {
          brightness += image[y][x];
        }
      }
      if(brightness > max) {
        max = brightness;
        coords = [i, j]
      }
    }
  }
  return coords;
}
